server/otlp/server.go

```diff
@@ -18,12 +18,12 @@ type Server struct {
 	pb.UnimplementedTraceServiceServer
 
 	addr string
-	db   model.RunRepository
+	db   model.Repository
 
 	gServer *grpc.Server
 }
 
-func NewServer(addr string, db model.RunRepository) *Server {
+func NewServer(addr string, db model.Repository) *Server {
```

Comment:
We are passing the whole db model to the constructor function from the `app.go` file so I just changed the type to have access to the `DefaultDataStore` function